packages/retail-ui-extensions/src/components/shared/CoreInputField.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * Represents an action type for the text field components.
+ * @typedef {Object} InputAction
+ * @property {string} label - The text displayed in the button.
+ * @property {boolean} [disabled] - Whether the button can be pressed.
```

Comment:
```suggestion
 * @property {boolean} [disabled] - Whether the button is disabled.
```

(Because it's actually the inverse of whether it can be pressed... Bit nitpicky but yeah!)